src/components/breadcrumbs/breadcrumbs.a11y.tsx

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+/// <reference types="../../../cypress/support"/>
+
+import React from 'react';
+import { EuiBreadcrumbs } from './breadcrumbs';
+
+const breadcrumbs = [
+  {
+    text: 'Animals',
+    href: '#',
+  },
+  {
+    text: 'Metazoans',
+    href: '#',
+  },
+  {
+    text: 'Chordates',
+    href: '#',
+  },
+  {
+    text: 'Vertebrates',
+    href: '#',
+  },
+  {
+    text: 'Tetrapods',
+    href: '#',
+  },
+  {
+    text: 'Reptiles',
+    href: '#',
+  },
+  {
+    text: 'Boa constrictor',
+    href: '#',
+  },
+  {
+    text: 'Nebulosa subspecies',
+  },
+];
+
+beforeEach(() => {
+  // Displays all breadcrumbs except the single truncated one
+  cy.viewport(768, 1024); // ipad-2
```

Comment:
very weakly held 2c: I don't see these comments are super useful personally, as our EUI breakpoints matter more than the specific device used. Maybe "small/medium/large breakpoint" instead?

```suggestion
  cy.viewport(768, 1024); // medium breakpoint
```

\+ expand this comment to all other instances of device comments across all 3 files